change-control-lambda/test/time-window.test.ts

```diff
@@ -30,6 +30,15 @@ DTEND:20190204T075900Z
 SUMMARY:Block3
 END:VEVENT
 
+BEGIN:VEVENT
+RRULE:FREQ=WEEKLY;INTERVAL=1
+DTEND:20200504T170000Z
+SUMMARY:Block4
+DTSTAMP:20200501T163641Z
+DTSTART:20200501T220000Z
+SEQUENCE:0
+END:VEVENT
```

Comment:
Would prefer if you could create a separate ics input data for these tests so that it's easier to reason when reading and reasoning about the test post-hoc.

Along with that also add comments what this recurrence in plain english (I'm not familiar with the ics format).

Reply:
Will do!